src/tests/TaxToken.t.sol

```diff
@@ -308,94 +308,115 @@ contract TaxTokenTest is Utility {
     // ~ burn() Testing ~
 
     // Test burn() from admin
-    function test_burn() public {
-        taxToken.transferOwnership(address(god));
-        assert(god.try_mint(address(taxToken), address(god), 10 ether));
+    function test_taxToken_burn() public {
+        taxToken.transferOwnership(address(dev));
+        assert(dev.try_mint(address(taxToken), address(dev), 10 ether));
 
         // Pre-state check.
-        assertEq(taxToken.balanceOf(address(god)), 10 ether);
+        assertEq(taxToken.balanceOf(address(dev)), 10 ether);
         assertEq(taxToken.totalSupply(), 1010 ether);
 
         // Burn 10 tokens to admin.
-        assert(god.try_burn(address(taxToken), address(god), 10 ether));
+        assert(dev.try_burn(address(taxToken), address(dev), 10 ether));
 
         //Post-state check.
-        assertEq(taxToken.balanceOf(address(god)), 0 ether);
+        assertEq(taxToken.balanceOf(address(dev)), 0 ether);
         assertEq(taxToken.totalSupply(), 1000 ether);
     }
 
-    function test_industryBurn_noLocked() public {
-        taxToken.transferOwnership(address(god));
-        assert(god.try_mint(address(taxToken), address(god), 10 ether));
-        assertEq(taxToken.industryTokens(address(god)), 0);
+    function test_taxToken_industryBurn_noLocked() public {
+        taxToken.transferOwnership(address(dev));
+        assert(dev.try_mint(address(taxToken), address(dev), 10 ether));
```

Comment:
Although I suppose it doesn't matter I think we should try to mint industry wallets to non-admin roles since there will never be a case that an admin should get industry tokens.